terraform/vars/adoagent-stg.tfvars

```diff
@@ -16,9 +16,6 @@ vm_username = "vhadoagent"
 # Network Security Group
 nsg_name = "vh-infra-core-ado-nsg"
 
-# environment
-env = "stg"
```

Comment:
I have seen this var used in your code, it will fail in staging if you remove this

Reply:
it is coming from the params of the run now so not needed there now